packages/backend/media-optimiser/src/media-optimiser.ts

```diff
@@ -0,0 +1,49 @@
+import sharp from "sharp";
+
+export interface MediaOptimiserAttributes {
+  fileName: string;
+  fileBuffer: Buffer;
+  formats?: Array<keyof sharp.FormatEnum> | sharp.AvailableFormatInfo[];
+  sizes?: number[];
+}
+
+export interface OptimisedMedia {
+  key: string;
+  optimisedImage: Buffer;
+  format: keyof sharp.FormatEnum | sharp.AvailableFormatInfo;
```

Comment:
Does this list change, depending on the version of libvips it's using under the hood? For example, heic is only supported if you have libheif installed.

Reply:
Yes that's correct